src/App.tsx

```diff
@@ -1,24 +1,256 @@
+  // total questions
+  const [totalQuestions, setTotalQuestions] = useState<any>(5)
+  // set difficulty
+  const [difficulty, setDifficulty] = useState<string>('easy');
+  // set category
+  const [category, setCategory] = useState<string>('9');
+
+
+  const trueSettings = () => {
+    setSettings(true);
+  }
+
+  const startTrivia = async ( ) => {
+    setScore(0);
+    setSettings(false);
+    setLoading(true);
+    setGameOver(false);
+
+    const newQuestions = await fetchQuizQuestions(
+      totalQuestions,
+      difficulty,
+      category
+    );
+
+    setQuestions(newQuestions);
+    setUserAnswers([]);
+    setNumber(0);
+    setLoading(false);
+
+  };
+
+  const checkAnswer = (e: React.MouseEvent<HTMLButtonElement>) => {
+    if(!gameOver){
+      //get the chosen answer
+      let answer = e.currentTarget.value;
+      // compare the chosen answer with the correct answer
+      let correct = answer === questions[number].correct_answer;
+      // if its correct, set score
+      if(correct) {
+        setScore(prev => prev + 1)
+        setCorrectAnswer(answer);
+        // set chosen button to true
+      } else {
+        // to give the  wrong answer a green background
+        setCorrectAnswer(questions[number].correct_answer);
+
+        // to give the  wrong answer a red background
+        setWrongAnswer(answer)
+      }
+
+
+      // save answer in answerObject
+      // save answer in object
+      const AnswerObject = {
+        question : questions[number].question,
+        answer,
+        correct,
+        correctAnswer : questions[number].correct_answer
+      }
+      // add answer above to userAnswer
+      setUserAnswers((prev) => [...prev, AnswerObject]);
+
+    }
+  };
+
+  const nextQuestion = () => {
+
+    if(number + 1 === parseInt(totalQuestions, 10)){
+      setGameOver(true);
+    }
+    else {
+      setNumber(prev => prev + 1); 
+    }
+  }
+
+  const settingsForm = (e: React.MouseEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    startTrivia();
+  }
+  
+  const handleDifficultyChange = (e: any) => {
+    setDifficulty(e.currentTarget.value)
+  }
+
+  const handleCategoryChange = (e: any) => {
+    setCategory(e.currentTarget.value)
+  }
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="quiz-app">
+      <h1> QUIZ APP </h1>
+
+      {
+        gameOver || (number + 1 === parseInt(totalQuestions, 10)) ?
+        <div className="start-settings">
+          {
+            !settings &&
+            <button className="start" onClick={startTrivia}> start </button>
```

Comment:
If you want to put some space around the text, use the CSS `padding` rule and not whitespace.